### ISSUE TITLE
Is it possible to add some customized constraints in Rkeras

### ISSUE THREAD

**Issue by z888888861** [rstudio/keras#255]
I am building a RNN model, besides the loss function, could I add an extra constraints in the model ?

My output is Y1 Y2....Yn, loss function make it minimize the MAPE. Meanwhile, I would like to minimize the error between any 4 consecutive Yhat and Y.  (Y is real, Yhat is predict)

Ex: (Y1-Yhat1)+(Y2-Yhat2)+..+(Y-Yhat4)  or  (Y2-Yhat2)+..+(Y-Yhat4)+(Y5-Yhat5)

Thank you.
